sdk/client/src/verifying_client.rs

```diff
@@ -41,28 +47,24 @@ type StateProof = (
 );
 
 #[derive(Clone, Debug)]
-pub struct VerifyingClient {
+pub struct VerifyingClient<S: Storage> {
```

Comment:
You shouldn't need the bound on Storage on the type definition, only on the impl